pkg/api/apikey.go

```diff
@@ -109,3 +109,15 @@ func (hs *HTTPServer) AddAPIKey(c *models.ReqContext, cmd models.AddApiKeyComman
 
 	return response.JSON(200, result)
 }
+
+// AddAPIKey adds an additional API key to a service account
+func (hs *HTTPServer) AdditionalAPIKey(c *models.ReqContext, cmd models.AddApiKeyCommand) response.Response {
```

Comment:
Do we really need another endpoint for this. Cant we just use AddAPIKey directly ?

Reply:
I agree, but our spec wants separate endpoints for service account functionality, including this one.  This will probably get moved to the new Service accounts service when it is finished

Reply:
The spec asks for a separate endpoint for adding additional api keys